pyfar/plot/_interaction.py

```diff
@@ -333,13 +381,20 @@ def update(self, plot):
             self._x_values = ['log', 'linear']
             self._x_id = self._x_values.index(getattr(self, self._x_param))
             # y-axis
-            self._y_type = ['other', 'other', 'other', 'other', 'other']
+            self._y_type = ['other', 'other', 'other', 'other']
             self._y_param = 'unit_gd'
-            self._y_values = ['auto', 's', 'ms', 'mus', 'samples']
+            self._y_values = ['s', 'ms', 'mus', 'samples', 'auto', 'other']
```

Comment:
'auto' is not the first anymore. What is the reason for changing the order here?

Reply:
It's a beer. When we changed the behavior of the `unit` parameter for all plots showing time on any axis, we forgot to change it here. I noted an other copy past mistake, which I corrected anywhere: y_type and y_param should be of the same length.